### DISCUSSION TITLE
Cannot trigger a subscription

### DISCUSSION THREAD

**Discussion by ogirginc** [rmosolgo/graphql-ruby#5166] [Q&A]
Hi! I have a question about triggering subscriptions, as I am failing to understand why my current code does not work. Here are my types:

```ruby
module Types
  class SubscriptionType < Types::BaseObject
    field :chat_update, subscription: Subscriptions::ChatUpdate
    field :chat_updated, subscription: Subscriptions::ChatUpdated, deprecation_reason: 'Use chatUpdate'
end
```

When I try to trigger `AppSchema.subscriptions.trigger(:chat_updated, {}, Chat.first, scope: 1)`, it works. However, when I try to trigger the newly added `chat_update` as `AppSchema.subscriptions.trigger(:chat_update, {}, Chat.first, scope: 1)`, nothing happens on both my rails console and server logs.

When I go back and update the types like this:

```diff
module Types
  class SubscriptionType < Types::BaseObject
    field :chat_update, subscription: Subscriptions::ChatUpdate
-    field :chat_updated, subscription: Subscriptions::ChatUpdated
+    field :chat_updated, subscription: Subscriptions::ChatUpdate
end
```

And run the trigger as `AppSchema.subscriptions.trigger(:chat_updated, {}, Chat.first, scope: 1)`, it works!

So it feels like I am missing a step when I attempt to add a new subscription (`chat_update` in this case). Does something pop up in anyone's mind about what I might be doing wrong or missing?

### Additional Info

1. We use `GraphQL::AnyCable`.
2. I cannot see my new `chatUpdate` under the subscriptions:

```ruby
my-app(dev)> GraphQL::AnyCable.stats(include_subscriptions: true)
=> {:total=>{:subscription=>9, :fingerprints=>2, :subscriptions=>11, :channel=>9}, :subscriptions=>{"messageUpdatedchatId6"=>4, "1chatUpdated"=>5}}
```
3. Both `app/javascript/graphql/schema.graphql` and `app/javascript/graphql/schema.json` are generated correctly.

**Comment by rmosolgo** [comment]
Hi! I don't see anything particularly _wrong_ in your description above, it sounds right to me. 

 Since AnyCable doesn't have `chatUpdate`, are you sure that your initial subscription is successful? Maybe there's an error in the initial `subscription { ... }` operation which is preventing the subscription from being created. Can you confirm in your browser's network tab that the request is successful?
